libraries/common/components/SelectBox/components/Item/index.jsx

```diff
@@ -34,12 +36,21 @@ class SelectBoxItem extends Component {
   render() {
     const Wrapper = this.props.wrapper;
 
+    let props = null;
```

Comment:
This block is not needed. You can simply set the `ref` prop to the incoming prop: `ref={this.props.forwardRef}`. This will work just fine, because the forwardRef prop falls back to `null`.

Reply:
True. I could not find a doc if ref can be `null`. Simplified